Xamarin.Forms.Platform.WinRT/WindowsBasePlatformServices.cs

```diff
@@ -78,6 +78,10 @@ public virtual Assembly[] GetAssemblies()
 			if (!assemblies.Contains(thisAssembly))
 				assemblies.Add(thisAssembly);
 
+			Assembly xamlAssembly = typeof(Xamarin.Forms.Xaml.IMarkupExtension).GetTypeInfo().Assembly;
+			if (!assemblies.Contains(xamlAssembly))
+				assemblies.Add(xamlAssembly);
```

Comment:
did we already had a dependency on that assembly ? or are we adding a new one ?

also, using `typeof(XamlLoader)` means more than using `IMarkupExtension` and helps understand which assembly we are talking about